example/src/components/ArrowBoxExamples.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import ArrowBox from '../../../src/components/ArrowBox';
```

Comment:
Yeah, I was trying to separate the two as separate PR's, but ImagePicker needs ArrowBox.